__tests__/topics.test.js

```diff
@@ -0,0 +1,56 @@
+const request = require('supertest')
+const app = require("../app")
+const seed = require("../db/seeds/seed")
+const db = require("../db/connection")
+const testData = require("../db/data/test-data/index")
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+    return db.end
+})
+
+describe("GET /api/healthcheck", () => {
+    test("200: responds with an okay message", () => {
+        return request(app)
+        .get("/api/healthcheck")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.msg).toBe("all okay");
+        });
+    });
+});
+
+describe("GET /api/topics tests", () => {
+    describe('Happy path tests', () => {
+        test('200: responds with an okay message', () => {
```

Comment:
Although I believe you should merge this test with another if you decide not to then the descriptions should be more accurate as it doesn't respond with an okay message just expecting the status code.